web/less/base.less

```diff
@@ -159,7 +165,7 @@ textarea
                 position: absolute;
                 background-color: #9f9f9f;
                 padding-left: 1.25em;
-                left: -1.25emx;
+                left: -1.25em;
```

Comment:
there arent any nested lists in the nav anymore so this change is correct yet affects nothing